shared/actions/teams.tsx

```diff
@@ -1120,6 +1120,26 @@ const deleteChannelConfirmed = async (state: TypedState, action: TeamsGen.Delete
   return false
 }
 
+const deleteMultiChannelsConfirmed = async (
+  state: TypedState,
+  action: TeamsGen.DeleteMultiChannelsConfirmedPayload
+) => {
+  const {teamID, channels} = action.payload
+
+  for (const conversationIDKey of channels) {
+    await RPCChatTypes.localDeleteConversationLocalRpcPromise(
+      {
+        channelName: '',
+        confirmed: true,
+        convID: ChatTypes.keyToConversationID(conversationIDKey),
+      },
+      Constants.teamWaitingKeyByID(teamID, state)
```

Comment:
Might be nice to use a specific waiting key for channel deletion here, and then keep the modal open until we're done waiting for this.